Modules/Relationships/src/Relationships.Domain/DomainErrors.cs

```diff
@@ -52,4 +52,10 @@ public static DomainError CannotRequestReactivationWhenThereIsAnOpenReactivation
         return new DomainError("error.platform.validation.relationshipRequest.cannotRequestReactivationWhenThereIsAnOpenReactivationRequest",
             $"You cannot request reactivation when there is an open reactivation request.");
     }
+
+    public static DomainError NoRejectableReactivationRequestExists()
+    {
+        return new DomainError("error.platform.validation.relationshipRequest.noRejectableReactivationRequestExists",
+            "There is no pending reactivation request or you are not allowed to revoke it. A reactivation request can only be rejected by the identity that did not requested it.");
```

Comment:
```suggestion
            "There is no pending reactivation request or you are not allowed to revoke it. A reactivation request can only be rejected by the identity that did not request it.");
```

Same for 
* NoRevocableReactivationRequestExists
* NoAcceptableReactivationRequestExists (other PR)

Reply:
I was debating myself on that one and went with `requested` in the end...

Reply:
But "did not requested" doesn't make sense, does it? 😲

Reply:
You should fix the error message for NoRevocableReactivationRequestExists as well (as mentioned in my initial comment)

Reply:
Somehow my brain read Rejectable instead of Revocable....